src/App.js

```diff
@@ -18,7 +18,6 @@ function App() {
 				{/* redirect to Home if route not found */}
 				<Route path="*" element={<Navigate to="/" />} />
 			</Routes>
-			<Footer />
```

Comment:
Since you removed the background from the footer, does it still need to be placed individually in the different pages?

Reply:
Yes, I would think so since each footer has a different background according to this year's Figma design. If you have another implementation in mind, let me know.

Reply:
Hmm, I was considering using the [:has() pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:has), but there is still limited support in Firefox. We can include the footer component on individual pages for now, but I would claim it's not an ideal solution.

Reply:
Thanks for the suggestion. I'll add it to the cleanup issue list then.